index.html

```diff
@@ -46,8 +46,11 @@ <h3>
         </div>
         <div class="step" data-step="7">
           <h3>
-            potato dishes overview
+            French fries are king.
           </h3>
+          <p>
+            When the potato finally arrives onto the plates of consumers, Americans find themselves enjoying the product in all shapes and sizes. French fries are the most popular, closely followed by mashed potatoes and baked potatoes.
```

Comment:
Visuals all look good, only thing is this description is a bit reversed with the next visual though, maybe we swap the descriptions? (and make next one more focused on deeper dive or something) Let me know what you think